.github/workflows/build_linux.yml

```diff
@@ -41,6 +41,9 @@ jobs:
           - build_name: "PubSub Encryption (MbedTLS) Build & Unit Tests (gcc)"
             cmd_deps: sudo apt-get install -y -qq libmbedtls-dev
             cmd_action: unit_tests_encryption_mbedtls_pubsub
+          - build_name: "PubSub SKS Build & Unit Tests (gcc)"
+            cmd_deps: sudo apt-get install -y -qq libmbedtls-dev
+            cmd_action: unit_tests_pubsub_sks
```

Comment:
First of all, great work! Is there a special reason to have a dedicated github runner? I would preffer to integrate these tests into one of the exisitng encryption tests.

Reply:
Thank you for the review. With SKS support we will have two mechanims to set the security keys. If Encryption is ON then we have no problem but if Encryption and SKS is ON then the keys has to be set from the keystorage. Also in Pull model, which will be the next PR, it requires the keys to be fetched from SKS before writer/reader group can be set operational. When they are operational, check the current key in key storage and set the encrypting keys.